server/services/teamsSummaryCountService.ts

```diff
@@ -0,0 +1,154 @@
+import logger from '../../logger'
+import type { StrapiApiClient, RestClientBuilder } from '../data'
+import { Component, DataItem, Product, SingleResponse } from '../data/strapiApiTypes'
+import AlertsService from './alertsService'
+
+type TeamAlertSummary = {
+  products: Record<string, Record<string, number>>
+  total: number
+}
+
+export default class TeamsSummaryCountService {
```

Comment:
Possibly make this more generic to Teams? or Alerts? TeamsSummaryCount is a very specific name for a service?